app/views/shared/components/_tags_list.html.slim

```diff
@@ -0,0 +1,6 @@
+p
+  span.glyphicon.glyphicon-tag.glyphicon--right
+  - tags.each_with_index do |tag, i|
+    = link_to tag.name, root_path
```

Comment:
Eventually we're going to want to have this link to the search path instead of the `root_path`. I presume once you've set up the tagging <> ES stuff we can add that in.